src/platform/testing/unit/mocha.opts

```diff
@@ -1,10 +1,10 @@
+--require @babel/register
 --require src/platform/testing/unit/mocha-setup.js
 --require src/platform/testing/unit/enzyme-setup.js
 --require choma
 --require @babel/polyfill
 --require blob-polyfill
 --require isomorphic-fetch
---compilers js:@babel/register,jsx:@babel/register
```

Comment:
`compilers` is no long a valid option. also the order matters- our `mocha-setup` uses import and requires babel to run before it.